docs/scripting.md

```diff
@@ -9,47 +9,45 @@ Hubot out of the box doesn't do too much, but it is an extensible, scriptable ro
 
 There are three main sources to load scripts from:
 
+* scripts loaded from external __npm packages__ and specified in `external-scripts.json`
 * all scripts __bundled__ with your hubot installation under `scripts/` dir
 * __community scripts__ specified in `hubot-scripts.json` and shipped in the `hubot-scripts` npm package
-* scripts loaded from external __npm packages__ and specified in `external-scripts.json`
```

Comment:
This order was to reflect the order the hubot loads them, not the order we recommend people use.